poc_mobile_verifier/src/error.rs

```diff
@@ -32,6 +32,12 @@ pub enum Error {
     MetaError(#[from] db_store::MetaError),
     #[error("join error")]
     JoinError(tokio::task::JoinError),
+    #[error("time out of range error")]
+    OutOfRangeError,
```

Comment:
Perhaps useful to have `OutOfRangeError(String)` and then update the error msg with the value which made it out of range?

Reply:
I don't think this error will ever occur, and unfortunately I have no way of actually typing the error type. I'd like to fix this properly by using `anyhow`